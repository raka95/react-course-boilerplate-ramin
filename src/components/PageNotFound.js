import {Link,NavLink} from  'react-router-dom';
import React from 'react';
export const PageNotFound =()=>(
    <div>
    404!- <Link to='/'>Go home</Link>
    <a href='/'>Go</a>
    </div>
);
export default PageNotFound;