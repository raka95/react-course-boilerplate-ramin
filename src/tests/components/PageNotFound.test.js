import React from 'react';
import PageNotFound from '../../components/PageNotFound';
import {shallow} from 'enzyme';

test("should render PageNotFound page",()=>{
    const wrapper=shallow(<PageNotFound />);
    expect(wrapper).toMatchSnapshot();
});