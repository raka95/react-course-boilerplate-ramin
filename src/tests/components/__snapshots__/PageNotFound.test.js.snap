// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PageNotFound page 1`] = `
<div>
  404!- 
  <Link
    replace={false}
    to="/"
  >
    Go home
  </Link>
  <a
    href="/"
  >
    Go
  </a>
</div>
`;
